core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataTest.scala

```diff
@@ -120,32 +127,48 @@ class GroupMetadataTest extends JUnitSuite {
   }
 
   @Test(expected = classOf[IllegalStateException])
+  def testEmptyToStableIllegalTransition() {
+    group.transitionTo(Stable)
```

Comment:
oh - i removed that as i added it when i did the `InitialRebalance` work. But i think this is fine as there is only a single line that could possibly throw the exception